src/main/java/com/jcabi/github/wire/CarefulWire.java

```diff
@@ -72,6 +73,7 @@
 @Immutable
 @ToString
 @EqualsAndHashCode(of = { "origin", "threshold" })
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
Otherwise, qulice complains about the repetition of the `NotNull` message:
```
[ERROR] /src/main/java/com/jcabi/github/wire/CarefulWire.java[107-107]: The String literal "response can't be NULL" appears 4 times in this file; the first occurrence is on line 107 (AvoidDuplicateLiterals)
```